python_env.yml

```diff
@@ -0,0 +1,16 @@
+name: timelapse_env
+
+channels:
+- conda-forge
+
+dependencies:
+- conda-forge::python=3.9
```

Comment:
Minor consideration: would it make sense to bump this version to the next minor version of Python? This is mostly a question about sustainability and possible performance gains you might get from newer versions of Python. Python 3.9 will be the "lowest supported" version after October 2024, when Python 3.8 reaches end-of-life ([link](https://devguide.python.org/versions/)).

Reply:
Python 3.12 added!